### PR TITLE
Add registration screen

### PR DESCRIPTION
**Pull request by IcemanOmikron**
Added registration screen that connects to database.

### CODE REVIEW

**Review by ggpepp — APPROVED** [review]
Looks good to me

File: lib/main.dart [hunk 1]
```diff
@@ -128,13 +140,10 @@ class _MyHomePageState extends State<MyHomePage> {
...
-                    SizedBox(
-                      height: 35.0,
-                    ),
+                    SizedBox(height: 25.0,),
                     loginButton,
-                    SizedBox(
-                      height: 15.0,
-                    ),
+                    SizedBox(height: 65.0,),
+                    registerButton,
```
Comment (IcemanOmikron) [inline lib/main.dart:146]:
```suggestion
                    registerButton,
                   
                   /*For demo purposes, these bypass the loginButton*/
                  SizedBox(height: 30,),
                  Row(
                    mainAxisAlignment: MainAxisAlignment.end,
                    children: <Widget>[
                      RaisedButton(
                        padding: EdgeInsets.fromLTRB(0, 0, 0, 0),
                        child: Text('demo: S'),
                        onPressed: (){
                          Navigator.push(
                            context,
                            MaterialPageRoute(builder: (context) => StudentScreen()),
                          );
                        },
                      ),
                      RaisedButton(
                        padding: EdgeInsets.fromLTRB(0, 0, 0, 0),
                        child: Text('demo: M'),
                        onPressed: (){
                          Navigator.push(
                          context,
                          MaterialPageRoute(builder: (context) => MentorScreen()),
                          );
                        },
                      ),
                   ],
                  ),
```

Comment (IcemanOmikron) [inline lib/main.dart:146]: For demo and debugging, adds two buttons to bypass the login screen and its call to the database. 

```suggestion
                    registerButton,
                   
                   /*For demo purposes, these bypass the loginButton*/
                  SizedBox(height: 30,),
                  Row(
                    mainAxisAlignment: MainAxisAlignment.end,
                    children: <Widget>[
                      RaisedButton(
                        padding: EdgeInsets.fromLTRB(0, 0, 0, 0),
                        child: Text('demo: S'),
                        onPressed: (){
                          Navigator.push(
                            context,
                            MaterialPageRoute(builder: (context) => StudentScreen()),
                          );
                        },
                      ),
                      RaisedButton(
                        padding: EdgeInsets.fromLTRB(0, 0, 0, 0),
                        child: Text('demo: M'),
                        onPressed: (){
                          Navigator.push(
                          context,
                          MaterialPageRoute(builder: (context) => MentorScreen()),
                          );
                        },
                      ),
                   ],
                  ),
```

File: lib/main.dart [hunk 2]
```diff
@@ -128,13 +140,38 @@ class _MyHomePageState extends State<MyHomePage> {
...
+                   
+                   /*For demo purposes, these bypass the loginButton*/
+                  SizedBox(height: 30,),
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.end,
+                    children: <Widget>[
+                      RaisedButton(
```
Comment (chwerth) [inline lib/main.dart:153]: These two raised buttons are basically the same, can we pull this out into one widget definition that's called twice?

**Review by chwerth — APPROVED** [review]
Cool, nice work

**Pull request merged**